content/en/docs/overview/trust-model/trust_model_personas.md

```diff
@@ -0,0 +1,211 @@
+This persona designs and creates the orchestration objects comprising the solution (e.g.
+kubernetes pod descriptions etc). These objects reference containers published by Container Image Providers.
+In some cases the Workload and Container Image Providers may be the same entity.
+The solution defined is intended to provide the Application or Workload which in turn provides
+value to the data owners (customers and clients).
+The workload provider and data owner could be part of same company/organisation but
+following the least privilege principle the workload provider should not be able to view or
+manipulate end user data without informed consent.
+- Need to prove to customer aspects of compliance.
+- Defines what the solution requires in order to run and maintain compliance (resources, utility
+  containers/services,  storage).
+- Chooses the method of verifying the container images (from those supported by Container Image
+  Provider) and obtains artifacts needed to allow verification to be completed within
+  the TEE.
+- Provide the boot images initially required by the TEE during
+  initialisation or designates a trusted party to do so.
+- Provide the attestation verification service, or designate a trusted party to provide the
+  attestation verification service.
+
+Example : 3rd party software vendor, cloud solution provider
+
+### Container Image Provider
+
+This persona is responsible for the part of the supply chain that builds container images and
+provides them for use by the solution. Since a workload can
+be composed of multiple containers, there may be multiple container image providers, some
+will be closely connected to the workload provider (business logic containers), others more
+independent to the workload provider (side car containers). The container image provider is expected
+to use a mechanism to allow provenance of container image to be established when a
+workload pulls in these images at deployment time. This can take the form of signing or encrypting
+the container images.
+- Builds container images.
+- Owner of business logic containers. These may contain proprietary algorithms, models or secrets.
+- Signs or encrypts the images.
+- Defines the methods available for verifying the container images to be used.
+- Publishes the signature verification key (public key).
+- Provides any decryption keys through a secure channel (generally to a key management system
+  controlled by a Key Broker Service).
+- Provides other required verification artifacts (secure channel may be considered).
+- Protects the keys used to sign or encrypt the container images.
+
+It is recognised that hybrid options exist surrounding workload provider and container provider.
+For example the workload provider may choose to protect their supply chain by
+signing/encrypting their own container images after following the build patterns already
+established by the container image provider.
+
+Example : Sidecar Container Provider
```

Comment:
that doesn't seem to be a tangible example for Container Image Provider

Reply:
Are you looking for an adjective or description that makes this more concrete (i.e. "tangible"), e.g. "Monitoring Sidecar Container Provider"?

Reply:
ah, what I meant is: this seems like a category that would itself need to be exemplified, i.e. I don't really know what a "(Monitoring) Sidecar Container Provider" would be. is that istio, linkerd, dockerhub or whoever owns the supplychain for a sidecar container?

Reply:
Oh, I see. Your examples make sense, and I can put istio here.